languages/erlang/exercises/concept/README.md

```diff
@@ -0,0 +1,25 @@
+# Erlang concept exercises
+
+The concept exercises are based on this [list of concepts][reference-shared].
+
+## Implemented exercises
+
+These are the concept exercises that have currently been implemented, as well as the concepts they teach and their prerequisite concepts:
+
+| exercise                                                            | concepts                                   | prerequisites                |
+| ------------------------------------------------------------------- | ------------------------------------------ | ---------------------------- |
+| [`basics`][concept-exercise-basics]                                 | `basics`                                   | -                            |
+
+It's only important that it's reasonably easy to _find_ the exercise. It's okay if the name isn't perfect. We **will** iterate on this.
+
+## TODO
+
+Thanks for wanting to contribute to the Erlang track's concept exercises! Contributions are very welcome!
+
+To contribute, please find and work on one of the [new exercise issues][issues-new-exercise] or [improve exercise issues][issues-improve-exercise].
+
+[reference-shared]: ../../reference/README.md
+[reference]: ./reference.md
+[concept-exercise-basics]: ./basics/.meta/design.md
```

Comment:
Broken link.

You don't use this link anywhere, why is it here?

Reply:
The link is used in the table. But it is broken since that exercise in the table doesn't exist in the repo.

Reply:
```suggestion
```